understanding/20/audio-description-prerecorded.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Audio Description (Prerecorded)</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Videos can be played with audio descriptions</dd>
-		<dt>Author task</dt><dd>Provide a synchronized spoken description of the visual content in videos</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Videos can be played with audio descriptions.</dd>
+         <dt>What to do</dt><dd>Provide a synchronized spoken description of the visual content in videos.</dd>
+         <dt>Why it's important</dt><dd>People who are blind or have low vision can hear about visual content while playing videos.</dd> 
```

Comment:
> What to do: Provide a synchronized spoken description of the visual content in videos.
> Why it's important: People who are blind and some with low vision can't see the visual content.

Reply:
I think I prefer the existing. I like differentiating between the synchronized version here (what you get when you're playing the video) and static forms elsewhere. We already have the "visual content" in the what to do.

Reply:
I agree. "can hear about visual content while playing videos." is more to the point.